src/components/Dialog/SettingDialog/SettingDialog.vue

```diff
@@ -298,39 +298,44 @@
               </QCardActions>
 
               <FileNamePatternDialog
-                v-model:open-dialog="showsFilePatternEditDialog"
+                v-model:open-dialog="showAudioFilePatternEditDialog"
+                v-model:fileNamePattern="audioFileNamePattern"
+                :defaultTemplate="DEFAULT_AUDIO_FILE_BASE_NAME_TEMPLATE"
+                :availableTags="[
+                  'index',
+                  'characterName',
+                  'styleName',
+                  'text',
+                  'date',
+                  'projectName',
+                ]"
+                :buildFileName="buildAudioFileNameFromRawData"
```

Comment:
これちょっと知らないのですが、reactとかでコンポーネントやフックに関数渡す場合ってどういう名称にするのが一般的なんですかね？
builderとかにするのどうかと提案しようと思ったのですが、そういえば知らないなと。

Reply:
これ調べてたんですが、Vueのcomputedはgetter、ReactのuseMemoはcalculateValueでした。
一貫性なし！

コード全体内で統一されていたら良さそう。
https://github.com/VOICEVOX/voicevox/blob/e49a2fcca8b680245370db4159e503506545ea0b/src/composables/useFetchNewUpdateInfos.ts#L11
で`currentVersionGetter`とあるので、動詞+erにするのとかどうでしょうか。
```suggestion
                :fileNameBuilder="buildAudioFileNameFromRawData"
```
（もちろん関数の方は動詞始まりで問題なさそう）

まあ、コード全体をクリーンに保つのに興味あれば 🙏